samples/server/pom.xml

```diff
@@ -128,8 +128,235 @@
           </execution>
         </executions>
       </plugin>
-
     </plugins>
   </build>
 
-</project>
+  <dependencyManagement>
+   <dependencies>
+      <dependency>
+        <groupId>asm</groupId>
+        <artifactId>asm</artifactId>
+        <version>4.2</version>
+        <scope>provided</scope>

```

Comment:
These dependencies should be handled in another way.

Having "provided" all around makes no sense, we are polluting the pom.xml unnecesary.

If these dependencies are added "by default" because of the **core** module then we have to stop adding them, at least by default.

One approach could be adding them on the **core** only if a profile (name it boot, dev, embedded) is active and it can be active by default). But when deploying to a WAR/EAR that profile should be deactivated.